install_files/ansible-base/roles/app/handlers/main.yml

```diff
@@ -19,6 +19,11 @@
     name: securedrop_worker
     state: present
 
+- name: restart apparmor
```

Comment:
It would be nice to comment here that this must be before the haveaged restart, and on the haveaged that it must be after the apparmor restart.

Reply:
Good point, fixed